subsys/bluetooth/mesh/vnd/Kconfig

```diff
@@ -68,7 +68,7 @@ config BT_MESH_DM_SRV_REFLECTOR_DELAY
 	int "Reflector start delay"
 	default 0
 	help
-	  Distance measurement reflector role start delay in milliseconds.
+	  Distance measurement reflector role start delay in microseconds.
 
 config BT_MESH_DM_SRV_INITIATOR_RANGING_WINDOW_US
```

Comment:
I just noticed one thing, and wonder if you would mind fixing while you first are editing this file..?
micro seconds -> microseconds for these two last config options.
Otherwise, LGTM :)